dist/index.html

```diff
@@ -4,10 +4,26 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/style.css">
```

Comment:
## Fix the styling bit.
- [ ] There are two ways to go about this. We can minimise the `css` in the root of the directory using `webpack` which I strongly recommend, or we can directly reference the already raw `CSS` in the root of the directory. As you are already using this latter option, the change will only require you reference the actual file. Right now, it can't get to it as the referencing is a bit off. See my suggestion 👇🏾 
### Option One
- [ ] prepend the referencing with two periods.
```suggestion
  <link rel="stylesheet" href="../style.css">
```

### Option Two
- [ ] You can add these changes in the `webpack.config.js` file to have it minimise both the images and the `CSS` files or you. You will need to add to a list of dependencies, the `loaders` used here. good luck Delice. I know you got this 💪🏾 

```javascript
const path = require('path');

module.exports = {
  mode: 'production',
  entry: './src/index.js',
  output: {
    filename: 'main.js',
    path: path.resolve(__dirname, 'dist'),
  },
  module: {
    rules: [
      {
        test: /\.css$/i,
        use: ['style-loader', 'css-loader'],
      },
      {
        test: /\.(png|svg|jpg|jpeg|gif)$/i,
        type: 'images',
      },
    ],
  },
};
```